atcoder-problems-frontend/src/pages/Internal/VirtualContest/ShowContest/FirstAcceptedCell.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { UserId } from "../../../../interfaces/Status";
+import { formatDuration } from "../../../../utils/DateUtil";
+import { UserNameLabel } from "../../../../components/UserNameLabel";
+
+export interface FirstAccepted {
+  userId: UserId;
+  time: number;
+}
+
+interface FirstAcceptedCellProps {
+  fastest: FirstAccepted | undefined;
+  showRating: boolean;
+}
+
+export const FirstAcceptedCell: React.FC<FirstAcceptedCellProps> = (props) => {
+  const { fastest, showRating } = props;
+
+  if (fastest) {
+    return (
+      <td>
+        <p className="font-weight-bold m-0 text-center">
+          {<UserNameLabel userId={fastest.userId} showRating={showRating} />}
```

Comment:
```suggestion
          <UserNameLabel userId={fastest.userId} showRating={showRating} />
```
これはReactNodeなので、そのまま出して大丈夫だと思います。